README.md

```diff
@@ -1,13 +1,40 @@
+<h1 align="center">
+
 ![](assets/logo.png)
 
-[![NPM](https://img.shields.io/npm/v/@rocketseat/unform.svg)](https://www.npmjs.com/package/@rocketseat/unform)
-[![Build Status](https://travis-ci.org/Rocketseat/unform.svg?branch=master)](https://travis-ci.org/Rocketseat/unform)
+</h1>
+
+<h3 align="center">
+Create ReactJS uncontrolled form structures with nested fields, validations and much more! 🚀
+</h3>
+
+<p align="center">
+
+<img alt="npm" src="https://img.shields.io/npm/v/@rocketseat/unform.svg">
+
+<img alt="dependencies" src="https://david-dm.org/@rocketseat/unform.svg">
```

Comment:
Should we use that?

Reply:
Yeah... It seems that this badge is rather unstable.

There is also the fact that being careful if all dependencies are up-to-date is a pain...

Reply:
Fair enough, I'm removing this.